src/StringUtils.php

```diff
@@ -49,4 +49,9 @@ public static function convertCaseToLower(string $value) : string
     {
         return mb_convert_case($value, MB_CASE_LOWER, 'UTF-8');
     }
+
+    public static function countCharacters(string $value) : int
```

Comment:
I think this method is not necessary because other methods do something that is not as straight forward as this